ydb/mvp/oidc_proxy/oidc_protected_page_handler.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "oidc_protected_page_nebius.h"
+#include "oidc_protected_page_yandex.h"
+
+namespace NMVP {
+
+class TProtectedPageHandler : public NActors::TActor<TProtectedPageHandler> {
+    using TBase = NActors::TActor<TProtectedPageHandler>;
+
+    NActors::TActorId HttpProxyId;
+    const TOpenIdConnectSettings Settings;
+
+public:
+    TProtectedPageHandler(const NActors::TActorId& httpProxyId, const TOpenIdConnectSettings& settings)
+        : TBase(&TProtectedPageHandler::StateWork)
+        , HttpProxyId(httpProxyId)
+        , Settings(settings)
+    {}
+
+    void Handle(NHttp::TEvHttpProxy::TEvHttpIncomingRequest::TPtr event, const NActors::TActorContext& ctx) {
```

Comment:
We can place this method in cpp-file and remove `#include "oidc_protected_page_nebius.h"` and `#include "oidc_protected_page_yandex.h"` from this header for compilation speed